audio/listener.rs

```diff
@@ -31,15 +31,15 @@ impl AudioListenerNode {
     pub fn new() -> Self {
         Self {
             channel_info: Default::default(),
-            position_x: Param::new(0.),
-            position_y: Param::new(0.),
-            position_z: Param::new(0.),
-            forward_x: Param::new(0.),
-            forward_y: Param::new(0.),
-            forward_z: Param::new(-1.),
-            up_x: Param::new(0.),
-            up_y: Param::new(1.),
-            up_z: Param::new(0.),
+            position_x: Param::new_krate(0.),
```

Comment:
This doesn't seem correct. Where in the spec does this happen?

Reply:
https://webaudio.github.io/web-audio-api/#AudioParam-attributes

> If the panningModel is "HRTF", the setting of the automationRate for any AudioParam of the PannerNode is ignored. Likewise, the setting of the automationRate for any AudioParam of the AudioListener is ignored. In this case, the AudioParam behaves as if the automationRate were set to "k-rate".

Reply:
This should happen in the panning implementation, the panner node is ignoring the automationRate, the listener still has the automation rate